src/SFA.DAS.AssessorService.Application/Handlers/Staff/LearnerDetailHandler.cs

```diff
@@ -73,13 +73,36 @@ public async Task<LearnerDetail> Handle(LearnerDetailRequest request, Cancellati
                 OrganisationId = epao.EndPointAssessorOrganisationId,
                 CertificateLogs = logs,
                 FundingModel = learner?.FundingModel,
+                CompletionStatusId = learner?.CompletionStatus,
```

Comment:
Can we keep it as CompletionStatus so it's badly consistent 😁 
And name the other one CompletionStatusDescription or something like that?